### ISSUE TITLE
gazelle <0.35 no longer supported

### ISSUE THREAD

**Issue by jbedard** [stackb/rules_proto#367]
After https://github.com/stackb/rules_proto/commit/5d1fb6aa7f023e8248ed112c269114fa03ae5b94

Was this breaking change intentional with 3.2.0?

I'm unable to upgrade gazelle atm due to https://github.com/bazelbuild/bazel-gazelle/issues/1751 so also can't upgrade rules_proto...

**Comment by pcj** [comment]
Yes, this was mentioned in https://github.com/stackb/rules_proto/pull/357 and why I bumped the major version from `v2` to `v3`

**Comment by pcj** [comment]
Perhaps the rules_python gazelle extension is performing resolve in parallel?  Maybe that new resolve map in bazel_gazelle needs an RWLock
